pkg/distributor/distributor.go

```diff
@@ -70,7 +70,8 @@ var (
 var allowedLabelsForLevel = map[string]struct{}{
 	"level": {}, "LEVEL": {}, "Level": {},
 	"severity": {}, "SEVERITY": {}, "Severity": {},
-	"lvl": {}, "LVL": {}, "Lvl": {},
+	"severity_text": {},
```

Comment:
I think we should also add `SeverityText`. Some applications also use `severityText`.. Can you please include possible cases?

Reply:
yep, https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-severitytext